### ISSUE TITLE
0 type is int and needs float

### ISSUE THREAD

**Issue by donpdonp** [node-influx/node-influx#104]
my script is writing float data to influx, but 0 values are always interpreted as an int, causing an error. whats going on here?

```javascript
influx.writePoint('sites', 0.0, null, function(err, response) {
  if(err) { console.log(err) }
})
```

```
[Error: write failed: field type conflict: input field "value" on measurement "sites" is type int64, already exists as type float
]
```

**Comment by donpdonp** [comment]
influxdb 0.9.2 has issues with float. fixed by upgraing to 0.9.4.1

https://github.com/influxdb/influxdb/issues/3519

**Comment by eliperelman** [comment]
@donpdonp also, InfluxDB changed how to write integers in 0.9.3. This library is not entirely compatible with 0.9.3+ and I've found the [influent package](https://www.npmjs.com/package/influent) works really well.
